bin/snapshots

```diff
@@ -23,7 +23,7 @@ declare -a args
 
 while [[ $# -gt 0 ]] ; do
   case "$1" in
-    -b|--branch) branch=$2; shift 2 ;;
+    -b|--branch) BRANCH=$2; shift 2 ;;
```

Comment:
Seems that this was broken. We read arg `--branch` and assign its value to `branch` variable, but this is never read. 

I guess the intention was to override `BRANCH`